### ISSUE TITLE
Uploading pre-built apk documentation doesn't take flavors into account

### ISSUE THREAD

**Issue by Nimrodda** [Triple-T/gradle-play-publisher#662]
My project has a few flavors. Each flavor has its own app ID.
I wanted to publish pre-built APKs per each flavor. After reading the documentation section about [Uploading a pre-existing artifact](https://github.com/Triple-T/gradle-play-publisher#uploading-a-pre-existing-artifact) I understood that I need to execute the following command per each flavor's apk:
`./gradlew publishApk --artifact-dir path/to/flavor/apk`

But executing the command above actually tried to publish that specific apk to all flavors!
I could see that the flavor specific publishing task, i.e. `publishApkFlavorBuildType`, was executed where the first one which matched the app ID of the provided apk was successful while the rest obviously failed.

I decided to try specifying the flavor specific `publishApk` task with `--artifact-dir` option, e.g.: `./gradlew publishApkFlavorBuildType --artifact-dir path/to/flavor/apk` and 🎉 it worked! 

I'm not sure if this is by design or a bug. If it's by design, please make the documentation clearer regarding flavor publishing.

Thank you for your awesome work! 😄

**Comment by SUPERCILEX** [comment]
Right, I would say that's expected since `publishApk` is a lifecycle task and doesn't actually do anything. However, I think it makes sense to try and parse a prebuild APK, check its package, and noop if the current build variant doesn't match.
